doctrine/dbal.rst

```diff
@@ -21,67 +21,21 @@ makes it easy to execute queries and perform other database actions.
     Read the official Doctrine `DBAL Documentation`_ to learn all the details
     and capabilities of Doctrine's DBAL library.
 
-To get started, configure the database connection parameters:
+To get started, configure the ``DATABASE_URL`` environment variable in ``.env``:
 
-.. configuration-block::
+.. code-block:: text
 
-    .. code-block:: yaml
+    # .env
 
-        # app/config/config.yml
-        doctrine:
-            dbal:
-                driver:   pdo_mysql
-                dbname:   Symfony
-                user:     root
-                password: null
-                charset:  UTF8
-                server_version: 5.6
+    # customize this line!
+    DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?charset=utf8mb4&serverVersion=5.7"
```

Comment:
Should `charset=utf8mb4&serverVersion=5.7` be removed in this DATABASE_URL env var?

Reply:
Good catch!